### ISSUE TITLE
Language not changing on the android tv app

### ISSUE THREAD

**Issue by moca12341** [jellyfin/jellyfin-androidtv#4315]
### Describe the bug

The tv client doesn't match the language of my device / profile.
I want to set the tv app to Hebrew.
the android device display language is set to Hebrew , and the profile I'm singed into is also set to Hebrew in the web version.
there is no setting in the app to change language so I assume its supposed to follow system settings.
I see there is 99% translation complete status for hebrew and its works fine in the web and on android phone app version of jellyfin.
also the metadata is displayed in Hebrew correctly its just the display language thats not changing. 

### Application version

0.18.2 

### Where did you install the app from?

None

### Device information

mi box

### Android version

android 12

### Jellyfin server version

10.10.3

**Comment by nielsvanvelzen** [comment]
The app uses the system language so you'll need to change that. On newer Android versions you can change the language per app. It's all managed by the system though.

**Comment by nielsvanvelzen** [comment]
Thanks for trashing the issue, really shows your appreciation.

**Comment by moca12341** [comment]
> Thanks for trashing the issue, really shows your appreciation.

I very much appreciate the project and the people who put the time and effort to create it on their free time.  I don't take it for granted.
in the above post (I restored it) I wrote that my device system settings are already set to Hebrew.
I add at this time that the issue happens for me across multiple android tv devices that are different models. 

the reply you gave me ignored that , and the ticket was closed. I assumed this was an oversight so I thought to delete this ticket and maybe open a new one with more detailed description on my part.
I'm new on github and didn't mean to harm / annoy
